### PR TITLE
fix home directory and username for GSNAPL instance

### PR DESCRIPTION
**Pull request by cdebourcy**

### CODE REVIEW

File: aws_batch/pipeline.py [hunk 1]
```diff
@@ -886,28 +886,30 @@ def run_gsnapl_remotely(sample, input_fa_1, input_fa_2,
     execute_command("aws s3 cp %s %s/" % (gsnap_ssh_key_s3_path, REF_DIR))
     key_path = REF_DIR +'/' + key_name
     execute_command("chmod 400 %s" % key_path)
-    commands =  "mkdir -p /home/ec2-user/batch-pipeline-workdir/%s;" % sample
-    commands += "aws s3 cp %s/%s /home/ec2-user/batch-pipeline-workdir/%s/ ; " % \
-                 (sample_s3_output_path, input_fa_1, sample)
-    commands += "aws s3 cp %s/%s /home/ec2-user/batch-pipeline-workdir/%s/ ; " % \
-                 (sample_s3_output_path, input_fa_2, sample)
-    commands += " ".join(['/home/ec2-user/bin/gsnapl',
+    remote_home_dir = "/home/ubuntu"
```
Comment (yunfangjuan) [inline aws_batch/pipeline.py]: why don't you just do 
remote_work_dir = ""/home/ubuntu/batch-pipeline-workdir/%s" % sample  ?

Comment (cdebourcy) [reply aws_batch/pipeline.py]: need it in several different paths, see below

Comment (cdebourcy) [reply aws_batch/pipeline.py]: need it in several different paths, see below

Comment (yunfangjuan) [reply aws_batch/pipeline.py]: can probably do both 
remote_home_dir 
remote_work_dir 
to make the code more succint though.

File: aws_batch/pipeline.py [hunk 2]
```diff
@@ -16,7 +16,7 @@
 
 INPUT_BUCKET = 's3://czbiohub-infectious-disease/UGANDA'
 OUTPUT_BUCKET = 's3://yunfang-workdir/id-uganda'
-KEY_S3_PATH = 's3://cdebourcy-test/cdebourcy_7-19-17.pem'
+KEY_S3_PATH = 's3://cdebourcy-test/idseq-alpha.pem'
```
Comment (yunfangjuan) [inline aws_batch/pipeline.py]: since you are changing it, why not put is into a s3 directory which is not cdebourcy-test?

**Pull request merged**